selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java

```diff
@@ -85,6 +90,7 @@ public void createAndDeleteProjectTest() throws ExecutionException, InterruptedE
     newWorkspace.clickOnCreateButtonAndOpenInIDE();
 
     String dashboardWindow = seleniumWebDriverHelper.switchToIdeFrameAndWaitAvailability();
+    toastLoader.waitToastLoaderAndClickButton("Start");
```

Comment:
IMHO we can make separate method `waitToastLoaderAndClickStartButton()` to leave list of arguments an empty.